app/models/twitter_user.rb

```diff
@@ -327,7 +327,7 @@ def replied(options = {})
       result = mentions.map { |m| m.user }.map { |u| u.uid = u.id; u }
       (options.has_key?(:uniq) && !options[:uniq]) ? result : result.uniq { |u| u.id.to_i }
     else
-      ExTwitter.new.select_replied_from_search(search_results, options).map { |u| u.uid = u.id; u }
+      ExTwitter::Client.new.select_replied_from_search(search_results, options).map { |u| u.uid = u.id; u }
```

Comment:
__[rubocop]__
* Line is too long. [107/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)